pom.xml

```diff
@@ -318,7 +318,7 @@ under the License.
             <dependency>
                 <groupId>com.google.protobuf</groupId>
                 <artifactId>protobuf-java</artifactId>
-                <version>3.11.4</version>
+                <version>3.21.7</version>
```

Comment:
[query]

- why we are upgrading it to `3.21.7` instead of most recent available version is `3.21.12` ?
https://mvnrepository.com/artifact/com.google.protobuf/protobuf-java/3.21.12
Is there any specific reason to keep it to `3.21.7` ?

Reply:
`3.21.7` is used as this is the version currently used by Flink
https://github.com/apache/flink/blob/master/pom.xml#L158